tealiumlibrary/src/main/java/com/tealium/core/messaging/DispatchSendCallbacks.kt

```diff
@@ -1,14 +1,16 @@
 package com.tealium.core.messaging
 
+import com.tealium.remotecommands.RemoteCommand
+
 interface AfterDispatchSendCallbacks {
-    fun sendRemoteCommand(url: String)
+    fun sendRemoteCommand(handler: RemoteCommand.ResponseHandler, url: String)
```

Comment:
Does it make more sense to have just a single `RemoteCommandRequest` argument here?